### DISCUSSION TITLE
Sample codes for accessing terraform remote state files and use of lz_key

### DISCUSSION THREAD

**Discussion by pascalpeh** [aztfmod/terraform-azurerm-caf#1444] [General]
Hi,

I have trying out this standalone CAF modules but could not figure out how to access the terraform remote state files.

Is there any sample codes for accessing terraform remote state files created by another terraform-azurerm-caf modules (Standalone) and also how do we access remote state using the lz_key? Or must the landing zone be created/integrated using the Azure CAF landing zone modules (https://github.com/Azure/caf-terraform-landingzones)?

Thanks

**Comment by brk3** [comment]
I've created a small repo that demonstrates how to integrate with CAF and look up content from other landing zones. It's more heavily documented than the configs in CAF (most of the magic is in `locals.tf`). It might be of use to you: https://github.com/brk3/terraform-landingzone-template
